packages/appstore/src/constants/wallet-intro-content-config.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import WalletsImage from 'Assets/svgs/wallets';
+import { localize } from '@deriv/translations';
+
+type TWalletIntroContent = {
+    image: JSX.Element;
+    title: string;
+    description: string;
+    bullets: string[];
+};
+
+const getWalletIntroContent = (is_eu: boolean): TWalletIntroContent[] => [
+    {
+        image: <WalletsImage image={is_eu ? 'introducing_wallets_eu' : 'introducing_wallets'} />,
+        title: localize('Introducing Wallets'),
+        description: is_eu ? localize('A new way to manage your funds') : localize('A better way to manage your funds'),
+        bullets: [
+            is_eu ? localize('One Wallet for all your transactions') : localize('One Wallet, one currency'),
+            is_eu
+                ? localize('Keep track of your trading funds in one place')
+                : localize('A Wallet for each currency to focus your funds'),
+            !is_eu &&
+                localize('Get one Wallet, get several {{dash}} your choice', {
```

Comment:
Can you use Localize component instead. Since we passing a variable

Reply:
Done 😃 👍🏼